.github/workflows/publish-open-vsx.yml

```diff
@@ -0,0 +1,45 @@
+# Publish the extension to the open-vsx registry at https://open-vsx.org
+
+name: Publish Open VSX
+on:
+  workflow_dispatch:
+
+jobs:
+  publish:
+    name: Publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '10.18.1'
+
+      - name: Install dependencies
+        run: |
+          cd extensions/ql-vscode
+          npm install
+        shell: bash
+
+      - name: Build
+        run: |
+          cd extensions/ql-vscode
+          npm run build -- --release
+        shell: bash
+
+      - name: Prepare artifacts
+        id: prepare-artifacts
+        run: |
+          mkdir artifacts
+          cp dist/*.vsix artifacts
+          VSIX_PATH="$(ls artifacts/*.vsix)"
+          echo "::set-output name=vsix_path::$VSIX_PATH"
+
+      - name: Publish to Registry
```

Comment:
This seems to be the only new step. Could we instead include it in the existing release workflow? We can use [environments](https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/reviewing-deployments) to manage human approval for release runs if necessary.

Reply:
That could work. It would also be great to move the publish to the vscode registry to the workflow. Though, I need to figure out how to get the deploy keys for this.